projects/packages/forms/src/blocks/contact-form/components/jetpack-akismet-panel.js

```diff
@@ -0,0 +1,63 @@
+import { Button } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import PluginIntegrationPanel from './shared/plugin-integration-panel';
+
+const AkismetPanel = () => {
+	const adminUrl = window?.jpFormsBlocks?.defaults?.formsAdminUrl || '';
+
+	return (
+		<PluginIntegrationPanel
+			title={ __( 'Spam protection', 'jetpack-forms' ) }
+			pluginSlug="akismet"
+			pluginPath="akismet/akismet"
+			installText={ __( 'Install akismet', 'jetpack-forms' ) }
+			activateText={ __( 'Activate akismet', 'jetpack-forms' ) }
+			description={ createInterpolateElement(
+				__(
+					"Add one-click spam protection for your forms with <a>Akismet</a>. Simply install the plugin and you're set.",
+					'jetpack-forms'
+				),
+				{
+					a: (
+						<a
+							href="https://wordpress.org/plugins/akismet/"
+							target="_blank"
+							rel="noopener noreferrer"
+						/>
+					),
+				}
+			) }
+			tracksEventName="jetpack_forms_upsell_akismet_click"
+			initialOpen={ false }
+		>
+			<p>
+				{ createInterpolateElement(
+					__( 'Your forms are protected from spam with <a>Akismet</a>!', 'jetpack-forms' ),
```

Comment:
[Commented elsewhere](https://github.com/Automattic/jetpack/pull/42043#pullrequestreview-2643863679) earlier so copying it here too:

> I'm not entirely sure but we might need to also add "Is configured" check with API keys; I saw some code around which help check if Akismet is configured but I don't remember if the API key is required for Akismet to work.

That's fine as a follow-up tho.